apis/v1/httproute_types.go

```diff
@@ -196,8 +196,19 @@ type HTTPRouteRule struct {
 	// Filters define the filters that are applied to requests that match
 	// this rule.
 	//
-	// The effects of ordering of multiple behaviors are currently unspecified.
-	// This can change in the future based on feedback during the alpha stage.
+	// Wherever possible, implementations SHOULD implement filters in the order
+	// they are specified.
+	//
+	// Implementations MAY choose to implement this ordering strictly, rejecting
+	// any combination or order of filters that can not be supported. If implementations
+	// choose a strict interpretation of filter ordering, they MUST clearly document
+	// that behavior.
+	//
+	// To reject an invalid combination or order of filters, implementations SHOULD
+	// consider the Route Rule(s) with this configuration invalid. If all Route Rule(s)
```

Comment:
On further thought, I think the `(s)` was unnecessary. 
```suggestion
	// consider the Route Rules with this configuration invalid. If all Route Rules
```